runtime/compiler/control/J9Options.cpp

```diff
@@ -100,6 +100,7 @@ int32_t J9::Options::_profilerStackSize = 128;
 
 int32_t J9::Options::_smallMethodBytecodeSizeThreshold = 0;
 int32_t J9::Options::_smallMethodBytecodeSizeThresholdForCold = -1; // -1 means not set (or disabled)
+int32_t J9::Options::_smallMethodBytecodeSizeThresholdForJITServerAOTCache = 32; // 0 means disabled
```

Comment:
Yes, the 32 value was chosen based on experiments where load is applied to a single JVM. Further experiments with 8 JVMs in parallel show that a value of 0 is best. I am going to change the default value to 0.

Reply:
So if the default is 0, this heuristic is disabled by default. Based on your experiments, it seems to only be useful when the JITServer handles a small number of clients, which is the opposite of when the AOT cache is most useful. I'm wondering if we should try to distinguish the cases of "small" and "large" number of clients, rather than have this off-by-default knob that is unlikely to actually be used.